src/helpers/utilities.ts

```diff
@@ -408,3 +408,19 @@ export const fromWei = (number: BigNumberish): string =>
 export const delay = (ms: number): Promise<void> => {
   return new Promise(resolve => setTimeout(resolve, ms));
 };
+
+/**
+ * @desc Can omit only flattened key, will not work with 'key.someObj.value'
+ */
+export const omitFlatten = <T extends object, K extends keyof T>(
+  obj: T | null | undefined,
+  keysToOmit: K[] | K
+): Omit<T, K> => {
+  const keys = Array.isArray(keysToOmit) ? keysToOmit : [keysToOmit];
+  const keysToUse = keys.map(String);
```

Comment:
what's this?

Reply:
I tried to copy Lasha's behavior, it compared keys as it a string